inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java

```diff
@@ -514,12 +571,35 @@ public Boolean updateWithoutCheck(InlongStreamRequest request, String operator)
         // update stream fields
         updateField(groupId, streamId, request.getFieldList());
         // update stream extension infos
-        saveOrUpdateExt(groupId, streamId, request.getExtList());
+        List<InlongStreamExtInfo> extList = request.getExtList();
+        extList = packExtParams(request, extList);
+        saveOrUpdateExt(groupId, streamId, extList);
 
         LOGGER.info("success to update inlong stream without check for groupId={} streamId={}", groupId, streamId);
         return true;
     }
 
+    /**
+     * Pack extended attributes into {@link InlongStreamExtInfo} 
+     * @param request the request
+     * @param extList the extList from frontend
+     * @return the packed extParams
+     */
+    private List<InlongStreamExtInfo> packExtParams(InlongStreamRequest request,
```

Comment:
I don't think additional parameters can be added in this way.If there are more parameters later, the code will become very redundant.

Reply:
done